config/pipeline.yaml

```
@@ -1825,6 +1825,135 @@ scheduler:
     platforms:
       - qemu-x86
 
+  - job: rt-tests-cyclicdeadline
+    event: &kbuild-gcc-12-arm64-preempt_rt-node-event
+      channel: node
+      name: kbuild-gcc-12-arm64-preempt_rt
+      result: pass
+    runtime: *runtime-lava-collabora
+    platforms: &kbuild-gcc-12-arm64-preempt_rt-platforms
```

```suggestion
    platforms: &arm64-preempt_rt-platforms
```

Please trim other "platforms" anchors in a similar way